src/main/kotlin/com/terraformation/backend/email/EmailNotificationService.kt

```diff
@@ -359,6 +362,39 @@ class EmailNotificationService(
         roles = setOf(Role.Admin, Role.Owner, Role.Manager))
   }
 
+  @EventListener
+  fun on(event: ObservationNotScheduledNotificationEvent) {
+    val plantingSite =
+        plantingSiteStore.fetchSiteById(
+            event.plantingSiteId,
+            PlantingSiteDepth.Site,
+        )
+    val organizationId = parentStore.getOrganizationId(event.plantingSiteId)!!
+    // return if we don't have a TF contact to send email to
+    val user = getTerraformationContactUser(organizationId)
+    val organization =
+        organizationStore.fetchOneById(
+            plantingSite.organizationId, OrganizationStore.FetchDepth.Organization)
+
+    user?.let {
```

Comment:
I think `if (user != null)` would probably be a bit easier to read here than the `let` expression.